applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/status.js

```diff
@@ -0,0 +1,264 @@
+		body.appendChild(
+			E('h3', 'Remotehosts')
+		);
+		var remotehost_table = E('table', { 'class': 'table cbi-section-table' }, [
+			E('tr', { 'class': 'tr table-titles' }, [
+				E('th', { 'class': 'th' }, _('IP address')),
+				E('th', { 'class': 'th' }, _('Identifier'))
+			])
+		]);
+		
+		var remotehosttableentries =[];
+		for(var IPaddr in Remotehosts) {
+			remotehosttableentries.push([
+							IPaddr, Remotehosts[IPaddr]['id']
+						]);
+				}
+
+		cbi_update_table(remotehost_table, remotehosttableentries, E('em', _('No data')));
+		body.appendChild(remotehost_table);
+		
+/////////////////////////
+		body.appendChild(
+			E('h3', 'Client list')
+		);
+		var connectioninfo_table = E('table', { 'class': 'table cbi-section-table' }, [
+			E('tr', { 'class': 'tr table-titles' }, [
+				E('th', { 'class': 'th' }, _('IP & Interface name','Combination of IP and interface name in usteer overview')),
+				E('th', { 'class': 'th' }, _('BSSID')),
+				E('th', { 'class': 'th' }, _('SSID')),
+				E('th', { 'class': 'th' }, _('Frequency', 'BSS operating frequency in usteer overview')),
+				E('th', { 'class': 'th' }, _('N', 'Number of associated clients in usteer overview')),
+				E('th', { 'class': 'th' }, _('Noise', 'Channel noise in usteer overview')),
+				E('th', { 'class': 'th' }, _('Load', 'Channel load in usteer overview')),
+				E('th', { 'class': 'th' }, _('Max assoc','Max associated clients in usteer overview')),
+				E('th', { 'class': 'th' }, _('Roam src','Roam source in usteer overview')),
+				E('th', { 'class': 'th' }, _('Roam tgt','Roam target in usteer overview')),
+				E('th', { 'class': 'th' }, _('RRM NR mac','mac address in RRM neighbor report in usteer overview')),
+				E('th', { 'class': 'th' }, _('RRM NR hex string','broadcast hex string for in RRM neighbor report in usteer overview'))
+				
+			])
+		]);
+	
+		var connectioninfo_table_entries =[];
+		for(var wlan in Localinfo) {
+			connectioninfo_table_entries.push([
+							'<nobr>'+wlan+'</nobr>', 
+							Localinfo[wlan]['bssid'],
+							Localinfo[wlan]['ssid'],
+							Localinfo[wlan]['freq'],
+							Localinfo[wlan]['n_assoc'],
+							Localinfo[wlan]['noise'],
+							Localinfo[wlan]['load'],
+							Localinfo[wlan]['max_assoc'],
+							Localinfo[wlan]['roam_events']['source'],
+							Localinfo[wlan]['roam_events']['target'],
+							Localinfo[wlan]['rrm_nr'][0],
+							Localinfo[wlan]['rrm_nr'][2]
+						]);
+				}	
+		for(var wlan in Remoteinfo) {
+			connectioninfo_table_entries.push([
+							'<nobr>'+wlan+'</nobr>', 
+							Remoteinfo[wlan]['bssid'],
+							Remoteinfo[wlan]['ssid'],
+							Remoteinfo[wlan]['freq'],
+							Remoteinfo[wlan]['n_assoc'],
+							Remoteinfo[wlan]['noise'],
+							Remoteinfo[wlan]['load'],
+							Remoteinfo[wlan]['max_assoc'],
+							Remoteinfo[wlan]['roam_events']['source'],
+							Remoteinfo[wlan]['roam_events']['target'],
+							Remoteinfo[wlan]['rrm_nr'][0],
+							Remoteinfo[wlan]['rrm_nr'][2]
```

Comment:
This is producing an error in my system. It seems `Remoteinfo[wlan]['rrm_nr']` does not exist. I don't know why. It exists in the `Localinfo` but not in the `Remoteinfo`.

![image](https://github.com/openwrt/luci/assets/2673520/0449b1d6-b9fe-4d1f-baed-210fdd791dbf)

Reply:
ok weird, on my setup it gets filled in, is the "network" correctly filled in on all APs? Regardless let me add some error checking on that, if its "undefined" then i will skip printing